### ISSUE TITLE
I can not import drei

### ISSUE THREAD

**Issue by richi-coder** [pmndrs/drei#1507]
What is the solution to this ?

- `three` version:  0.153.0
- `@react-three/fiber` version: 8.13.2
- `@react-three/drei` version: 9.74.12
- `node` version: 19.6.0
- `npm` (or `yarn`) version: 9.4.0
- `vite` version: 4.3.9

### Problem description:

Once react-three/drei is installed, I get an error in my react-vite project when using this import:

### Relevant code:

```
import { Stats, OrbitControls } from "@react-three/drei"
```
The error is: *Failed to resolve entry for package "@react-three/drei". The package may have incorrect main/module/exports specified in its package.json.*

Please help!
My first time trying react-three/drei

**Comment by wass08** [comment]
I'm also facing this issue with Vite on a new project.
Copying the `package.json` content from a working project with `rm -rf node_modules yarn.lock` doesn't solve this issue. 
A real mystery

**Comment by zgid123** [comment]
I think it is because the new build does not contain the `index.js`, it only contains the `index.cjs.js`, so the `module` cannot be import since it is `"module": "index.js",`

**Comment by jksntaylor** [comment]
I manually changed the module path to ```"module": "index.cjs.js"``` and it seems to be working

**Comment by zgid123** [comment]
no, don't do that, cjs is `commonjs`, but `vite` is using `esm` as I remember, I think you should wait for a new deployment or use the version `9.74.11` at the moment

**Comment by wass08** [comment]
Does it work for you with drei version `9.74.11`?
I have this error as soon as I import anything from `drei`:
```
The plugin "vite:dep-scan" was triggered by this import

    src/App.jsx:1:30:
      1 │ import { OrbitControls } from "@react-three/drei";
```

I tried with those package versions without success:
```
{
    "@react-three/drei": "^9.74.11",
    "@react-three/fiber": "^8.13.0",
    "@types/three": "^0.152.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "three": "^0.153.0"
  }
```

**Comment by zgid123** [comment]
with your package json and yarn.lock is deleted, yarn will install the latest version `9.74.12`, can you try

```json
{
    "@react-three/drei": "9.74.11",
    "@react-three/fiber": "^8.13.0",
    "@types/three": "^0.152.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "three": "^0.153.0"
  }
```

**Comment by helgee** [comment]
> Is this still an issue with `9.74.13`?

Yes, unfortunately.

**Comment by drcmda** [comment]
@CodyJasonBennett @abernier is mediapipe so problematic? it seems to cause a lot of odd issues recently, there was another instant where windows avast suddenly flags drei as virulent though i do not know if that's related.

**Comment by CodyJasonBennett** [comment]
Yes, they pushed an RC with fixes that are supposed to fix compat here. A/Bing that with #1509.

**Comment by CodyJasonBennett** [comment]
Should be fixed with [`v9.74.14`](https://github.com/pmndrs/drei/releases/tag/v9.74.14).
